### ISSUE TITLE
Coord_flip and angles

### ISSUE THREAD

**Issue by dirmeier** [hms-dbmi/UpSetR#64]
Dear UpSetR developers,

maybe I just did not find the functionality, but, I think it would be nice to be able to flip axes. For plots with many bars, numbers on the bars get unreadable or too small. So having the bars on the x-axis and intersection size on the y axis would help.

Thanks and cheers!

**Comment by cbgargas** [comment]
I agree with Simon, either adding in a transpose option to upset()  or allowing us to transpose our data in upset() would be very handy.

Thanks for making such a useful package,  cheers!
